vm/src/stdlib/builtins.rs

```diff
@@ -146,20 +161,43 @@ mod builtins {
             }
             #[cfg(feature = "rustpython-parser")]
             {
+                use crate::builtins::PyMemoryView;
+                use crate::common::borrow::BorrowedValue;
+                use crate::protocol::PyBuffer;
+                use crate::types::AsBuffer;
                 use crate::{builtins::PyBytesRef, convert::ToPyException};
                 use num_traits::Zero;
                 use rustpython_parser as parser;
 
-                let source = Either::<PyStrRef, PyBytesRef>::try_from_object(vm, args.source)?;
+                //
+                let source =
+                    Either::<PyStrRef, Either<PyBytesRef, PyRef<PyMemoryView>>>::try_from_object(
```

Comment:
Does this fit in? This is comparably more recent concept we have about buffers.

```suggestion
                    ArgStrOrBytesLike::try_from_object(
```

Reply:
`ArgStrOrBytesLike` works